README.md

```diff
@@ -2,11 +2,9 @@
 
 ## Overview
 
->**CAUTION:** This repository is in a very early stage. Use it at your own risk.
-
 Kyma Control Plane (KCP) is a central system to manage Kyma Runtimes.
 
-For more information on KCP and its components, read the [documentation](https://kyma-project.io/docs/control-plane/).
+For more information on KCP and its components, read the [documentation](https://github.com/kyma-project/control-plane/tree/main/docs).
```

Comment:
```suggestion
For more information on KCP and its components, read the [KCP documentation](https://github.com/kyma-project/control-plane/tree/main/docs).
```
I like specific link titles, because if ever the link is broken, a meaningful title gives the reader the chance to find the doc by themselves.